dependencies.rb

```diff
@@ -0,0 +1,29 @@
+# Version definitions
+$AMPLIFY_VERSION = '1.0.1'
+$AWS_SDK_VERSION = '2.13.4'
+
+# http://guides.cocoapods.org/using/the-podfile.html#specifying-pod-versions
+def optimistic_version(pod_version)
+  "~> #{pod_version}"
+end
+
+# GitHub tag name for Amplify releases
+def release_tag
+  "v#{$AMPLIFY_VERSION}"
```

Comment:
@royjit  @palpatim  i recall we want to keep this 'v' here and update the circleCI workflows scripts to account for this, right?

Reply:
@lawmicha Correct

Reply:
Yes, the tag name should start with v